packages/create-docusaurus/templates/classic/package.json

```diff
@@ -16,11 +16,10 @@
   "dependencies": {
     "@docusaurus/core": "2.0.0-beta.18",
     "@docusaurus/preset-classic": "2.0.0-beta.18",
-    "@mdx-js/react": "^1.6.22",
```

Comment:
... Sorry, we are not interested in maintaining a fork of MDX. Please refrain from making these architectural changes without first discussing with us. Thanks!